### DISCUSSION TITLE
Getting assembly error while running sample projects

### DISCUSSION THREAD

**Discussion by ks1990cn** [dotnet/aspnetcore#59312] [General]
Hi,

I am getting error while running sample projects. I am getting same error for APIExplorer. 

I tried to load ApiExplorer assembly with dependency and its loading perfectly.

Can anyone help me with this?

<img width="931" alt="image" src="https://github.com/user-attachments/assets/7b0de367-f49b-49ae-b613-9b0c29a8a3b2">

**Comment by davidfowl** [comment]
Hard to debug a screenshot. Do you have a runnable sample that reproduces the problem?

**Reply by ks1990cn** [reply]
I setup my code & tried to run the sample application.

**Reply by ks1990cn** [reply]
I tried to build and load with dependencies for ApiExplorer but I still face this issue.

**Reply by davidfowl** [reply]
I'm not sure how to help with this limited amount of information.

**Reply by ks1990cn** [reply]
Hi @davidfowl , I was able to run sample projects. Below are things I did - 
- Built Mvc project.
- Reloaded project with dependencies for Mvc.ApiExplorer.
- Added in csproj for sample projects ` <Reference Include="Microsoft.AspNetCore.Mvc.ApiExplorer" />`
- Built project with sample like `src/http`

But I have question now, why building on my machine is special than anyones. Can  you check again if ApiExplorer project is required to run samples? 
Also Can you help me which nuget package source you are selecting?

**Reply by ks1990cn** [reply]
Hi, I was trying to run multiple projects. I get issue with Middleware - Session sample code. Taking look into it. Will let you know If I will be able to resolve it. Its related to DI. But I can run most of samples now.

<img width="383" alt="image" src="https://github.com/user-attachments/assets/fff00de1-e56e-4d81-a50b-1ade372b7e01">
